vortex-flatbuffers/flatbuffers/vortex-file/footer.fbs

```diff
@@ -1,34 +1,90 @@
 include "vortex-layout/layout.fbs";
 
-// [file layout]
-/// The `FileLayout` stores the root `Layout` as well as location information for each referenced segment.
-table FileLayout {
-    root_layout: Layout;
+// [postscript]
+/// The `Postscript` is guaranteed by the file format to never exceed
+/// 65528 bytes (i.e., u16::MAX - 8 bytes) in length, and is immediately
+/// followed by an 8-byte `EndOfFile` struct.
+table Postscript {
+    /// The encodings used within the file
+    encodings: [EncodingVTable];
```

Comment:
given that we have a not-that-large hard limit on Postscript size, probably makes more sense to put `encodings` and `layouts` in a a Segment each (that the writer may indeed write last, just before the Postscript)

Reply:
Yeah... but I'm kind of expect single digit layouts, and 10-20 encodings tops. How much can you fit in 65kb?!